content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -311,8 +311,8 @@ Use the `set`, `push` `updateMany` and `deleteMany` operations to change a list
 
 - `set`: Set an embedded list of composite types, overriding any existing list
 - `push`: Push values to the end of an embedded list of composite types
-  <!-- - `updateMany`: Update many composite types at once -->
-  <!-- - `deleteMany`: Delete many composite types at once -->
+- `updateMany`: Update many composite types at once -->
+- `deleteMany`: Delete many composite types at once -->
```

Comment:
I'm not sure we're actually going to deliver these by the end of this sprint. Will respond back.

Either way, be careful to remove the end of the HTML comment `-->`

Reply:
These won't make the release

Reply:
I've removed the `-->` while fixing the other formatting errors. Otherwise left this for you to comment out whatever doesn't make it to the release